### PR TITLE
Added quotes from Porter Robinson tracks

### PR DESCRIPTION
**Pull request by seyaidev**
Added quotes from Porter Robinson's "Everything Goes On" and "Something Comforting" to `english.json`

_From "Everything Goes On"_
`Don't try to make yourself remember, darling. Don't look for me, I'm just a story you've been told. So let's pretend a little longer. 'Cause when we're gone, everything goes on.`

`But if I was gone tomorrow, won't the waves crash on? Is it selfish that I'm happy as we pass the setting sun? Someday I'll be overcast but you won't have to cry. 'Cause we'll do the grieving while I'm by your side.`

_From "Something Comforting"_
`Take what you want, but you're tied by the tooth. Wasn't it meant to relieve you? So suffer the signs that you needed to change. Heed all the ways you'd been thinking.`

### CODE REVIEW

**Review by rizwanmustafa — APPROVED** [review]
As far as the syntax goes. Everything seems fine. It's now up to Miodec

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -35442,6 +35442,30 @@
...
+      "text": "Don't try to make yourself remember, darling. Don't look for me, I'm just a story you've been told. So let's pretend a little longer. 'Cause when we're gone, everything goes on.",
+      "source": "Everything Goes On",
+      "length": 177,
+      "id": 6228
+    },
+    {
+      "text": "Don't try to make yourself remember, darling. Don't look for me, I'm just a story you've been told. So let's pretend a little longer. 'Cause when we're gone, everything goes on.",
+      "source": "Everything Goes On",
+      "length": 177,
+      "id": 6229
```
Comment (Miodec) [inline frontend/static/quotes/english.json:35456]: Same quote twice?

**Review by Miodec — CHANGES_REQUESTED** [review]
Remove duplicate, also please update the sources to mention the author, for example `Porter Robinson - Everything Goes On`

File: frontend/static/quotes/english.json (hunk 1 above)
Comment (seyaidev) [reply frontend/static/quotes/english.json:35456]: Whoops, my bad.

**Comment by seyaidev** [comment]
Duplicate should be removed now.

**Review by Miodec — APPROVED** [review]

**Pull request merged**